### ISSUE TITLE
How to show tray applet on left click?

### ISSUE THREAD

**Issue by ervinpopescu** [korikori/ideapad-perf#4]
I'm not familiar with `ruby` and I would very much like **not** rewriting the applet just for a tiny modification. The online docs I found for the `ruby-gtk3` binding weren't that helpful, if you could tell me what I can follow in the future, that would be great too. 

From https://docs.gtk.org/gtk3/method.Menu.popup.html
>The button parameter should be the mouse button pressed to initiate the menu popup. If the menu popup was initiated by something other than a mouse button press, such as a mouse button release or a keypress, button should be 0.

This is the line I should change, right?
```
si.signal_connect('popup-menu'){|tray, button, time| menu.popup(nil, nil, button, time)}
```

Thanks in advance!

**Comment by korikori** [comment]
tldr: yes, IIRC if you want to change the right-click action to left-click, you should change `popup-menu` to `activate` in the quoted line.

I have to say I'm not the least bit surprised to receive this issue, as I found out the same thing when I set out to make this applet - there's very limited documentation on `ruby-gtk3` and I actually used this tutorial from 2010 for `ruby-gtk2` to get going:
http://ixti.net/development/ruby/2010/06/16/system-tray-icon-with-ruby-and-gtk2.html
So I had the idea to re-create the same tutorial but for `ruby-gtk3` with this applet... but then I never did :(

edit: just tried simply replacing `popup-menu` to `activate` and it crashes. I'll take a look into it later today.

**Comment by ervinpopescu** [comment]
It crashes for me too. Waiting for your reply!

**Comment by korikori** [comment]
I'm sorry, I've tried messing around, searched online for "left click popup menu gtk" in ANY language and just can't seem to find anything that works. GTK does not make any sense to me. I will keep on looking but I'm not very hopeful.

**Comment by ervinpopescu** [comment]
No problem! I "ported" the applet to `python` and it works perfectly so far.

**Comment by korikori** [comment]
That's great, feel free to make a pull request to include it as an option for others.

**Comment by ervinpopescu** [comment]
Will try, I'm really busy these days, for the next week and possibly more. Closing!

**Comment by ervinpopescu** [comment]
@korikori 

Quick note, I use the battery-only branch. It's fairly easy for me to do the same applet for the main branch. Do I include that too in the PR?

**Comment by korikori** [comment]
Sure, if it's not a problem - I'm certain that both would be useful to someone!
